fetch.bs

```diff
@@ -2169,6 +2174,15 @@ otherwise, it is unset.
 <a for=request>done flag</a>, and <a for=request>timing allow failed flag</a> are used as
 bookkeeping details by the <a for=/>fetch</a> algorithm.
 
+<p>A <a for=/>request</a> has an associated
+<dfn export for=request id=concept-request-id>request id</dfn> which is a unique
+string automatically set when the <a for=/>request</a> is created.
```

Comment:
I'd prefer if we call this just "id" or "WebDriver id".

Reply:
I went with just "id".